lib/cfd_trading/cfd_trading_change_notifier.dart

```diff
@@ -28,10 +27,14 @@ class CfdTradingChangeNotifier extends ChangeNotifier {
     super.notifyListeners();
   }
 
+  Future<void> listCfds() async {
+    cfds = await api.listCfds();
+    super.notifyListeners();
+  }
```

Comment:
It's a bit annoying that we have to call this every time we affect the list of CFDs. It would be nicer if it were just updated periodically (like we do below) and also every time we navigate to the page that lists the CFDs.

Reply:
Not dealing with it right now, because we want to move away from polling if possible.